packages/dashboard-core-plugins/src/panels/IrisGridPanel.tsx

```diff
@@ -562,6 +563,21 @@ export class IrisGridPanel extends PureComponent<
     })
   );
 
+  getWidgetPanelDescriptor = memoize(
+    (
+      metadata: IrisGridPanelProps['metadata'],
+      description?: string
+    ): WidgetPanelDescriptor => {
+      const name = getTableNameFromMetadata(metadata);
+      return {
+        ...metadata,
+        type: 'Table',
+        name,
```

Comment:
Same comment as Chart. Except here there's multiple types it could be. `Table`, `TreeTable`, `HierarchicalTable`, and `PartitionedTable`. But we just show `Table` in the tooltip for all of them

Reply:
`metadata` will override the type if it's set.